.changeset/moody-pears-suffer.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-circleci': patch
+---
+
+Fixed bug in circleci plugin. The bug fix is with respect to restartBuild action in circleci.Changed the vcstype from Git to map the vcstype dynamically.
```

Comment:
Ack sorry, the spelling and grammar check is complaining about a few words here. You can escape with backticks to get around it (or add words to the [vocab file](https://github.com/backstage/backstage/blob/master/.github/styles/vocab.txt)). Maybe update to something like:

```
Fixed a bug in the `CircleCI` plugin where restarting builds was hard-coded to GitHub rather than introspecting the entity source location.
```

Thank you for the fix! 🎉

Reply:
@timbonicus Thats good to know. Thank you! Updated the file.